test/dao/operational.ts

```diff
@@ -0,0 +1,27 @@
+// Declare imports
+import * as helpers from '../helpers/contract-helpers';
+import { expect } from 'chai';
+
+let ssvNetworkContract: any;
+
+// Declare globals
+describe('DAO operational Tests', () => {
+    beforeEach(async () => {
+        // Initialize contract
+        ssvNetworkContract = (await helpers.initializeContract()).contract;
+    });
+
+    it('starting the transfer process does not change owner', async () => {
```

Comment:
uppercase and format.
add one test about change onlyOwner variable and try to change with the old owner